5g-control-plane/templates/bin/_upf-adapter-run.sh.tpl

```diff
@@ -7,11 +7,14 @@
 set -xe
 
 {{- if .Values.config.coreDump.enabled }}
-cp /aether/upfadapter /tmp/coredump/
+cp /usr/local/bin/upfadapter /tmp/coredump/
 {{- end }}
 
-cd /aether
+CFGPATH=/home
+FILENAME=config.yaml
```

Comment:
all config files name has NF in it. e.g. amf config is amfcfg.yaml .. Can we not keep same for upf adapter?

Reply:
Done.